internal/ignition/ignition_test.go

```diff
@@ -119,6 +124,15 @@ var _ = Describe("Bootstrap Ignition Update", func() {
 				Expect(err).NotTo(HaveOccurred())
 				Expect(bmh.Annotations).To(HaveKey(bmh_v1alpha1.StatusAnnotation))
 			})
+			It("adds the marker file", func() {
+				var found bool
+				for _, f := range config.Storage.Files {
+					if f.Path == "/opt/openshift/assisted-install-bootstrap" {
```

Comment:
the current pattern is to use a file extension of done to indicate your component has completed its tasks on the bootstrap node. I would prefur if we maintained that.

https://github.com/openshift/installer/blob/ba6d7fe087eada5a4fc260064c85a484a8c45aaf/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L287

Reply:
Right, but that timing doesn't make sense for what we're using this for. This is exactly why I suggested not using the `.done` extension.

We need this file to be here before we've completed our work on the bootstrap. As an installer I would consider "our work" to be the entire bootstrap node lifecycle.

Reply:
technically `done` is a reference to the rendering of manifests by operator render commands via bootkube. your component does not perform those actions to my knowledge.

>We need this file to be here before we've completed our work on the bootstrap. As an installer I would consider "our work" to be the entire bootstrap node lifecycle.

That may be true but that is not the meaning of the file's existence. Each operator adds this file so that simple checks can exist to omit rerun of logic on systemd unit retry. We are just piggybacking the proccess and an observable condition.

Reply:
basically, why create a new pattern?

Reply:
> basically, why create a new pattern?

Because the file we're creating isn't serving the same purpose as it is in all the other cases. I think it's more confusing to conform to an existing pattern if none of the semantics about that pattern apply to our case.

If there's a better location or name for this file that would be more appropriate for what we're doing I'd be happy to move it.

Reply:
Your component is performing a no-op and thus is done in the context provided. This seems straightforward to me.. we will read whatever you write to disk I gave my reasoning.

Reply:
Don't know how this is piggybacking on the installer ".done" pattern.
Except for both use a file.
There is no operator, there is no rendering, and the file purpose is different. 
Seems that it's targeting etcd on the bootstrap node, nothing else.